pkg/controller/installation/validation_test.go

```diff
@@ -411,6 +411,24 @@ var _ = Describe("Installation validation tests", func() {
 		Expect(err).To(HaveOccurred())
 	})
 
+	It("should error on not-allowed CNI sysctl tuning plugin config", func() {
+		instance.Spec.CalicoNetwork.Sysctl = []operator.Sysctl{
+			{
+				Key:   "net.ipv4.tcp_keepalive_intvl",
+				Value: "15",
+			}, {
+				Key:   "net.ipv4.tcp_keepalive_probes",
+				Value: "6",
+			},
+			{
+				Key:   "net.ipv4.tcp_keepalive_time",
```

Comment:
These 4 keys currently match the "Allowed" tuning config, so I think this test is wrong. I'd suggest renaming this test to "should validate the allowed tuning plugin config" and adding another test with a Key that is not one of the allowed keys.